starter-code/index.js

```diff
@@ -1,11 +1,47 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
+  constructor() {
+    this.items  = [];
+    this.length = 0;
+  }
+  add(x){
+    this.items.push(x);
+    this.length++;
+  }
+
+  get(x){
+    if(this.items.length=== 0){
+      throw new Error("EmptySortedList")}
+    if(x > this.length){
+      throw new Error("OutOfBounds")}
+    return this.items[x];
+}
+
+  max(items){
+    if(this.items.length=== 0){
+      throw new Error("EmptySortedList")
+    }
+    return Math.max(...items);
+  }
+
+  min(items){
+    if(this.items.length=== 0){
+      throw new Error("EmptySortedList");
+    }
+    return Math.min(...items);
+  }
+
+  sum(items){
+    this.items.reduce(acc, curr)
+      return acc + curr;
+    }
+
+
+    average(){
```

Comment:
and you're close here. But you don't call the the sum function. To call the function you need to do 

```
 return this.sum() / this.length;
```

In order the pass the other test you just need to add a condition that checks to see if the length is 0